packages/run-protocol/src/vpool-xyk-amm/addPool.js

```diff
@@ -170,7 +169,8 @@ export const makeAddPoolInvitation = (
     zcf.reallocate(reserveLiquidityTokenSeat, seat);
     seat.exit();
     pool.updateState();
-    brandToLiquidityMint.delete(secondaryBrand);
+    // ??? WHY?
+    // brandToLiquidityMint.delete(secondaryBrand);
```

Comment:
I thought this would be a simple change but I spent some time trying to figure out why the issuer wasn't in the `brandToLiquidityMint` store. It gets deleted here.

@Chris-Hibbert why?

Reply:
I can't think of a reason to do that. It looks purely like a mistake. I can't think of what I might have intended to do there that would have been similar.